sparse/usr/sbin/flash-partition

```diff
@@ -146,6 +146,14 @@ if on_device; then
         # partition flashing as it is not needed.
         if test $1 = boot && grep data_subdir= /proc/cmdline &> /dev/null; then
           echo "Skipping flashing of boot partition as multirom setup detected."
+          echo "Update multirom boot image, ramdisk and kernel"
+          pref_path="/android"
+          [[ -d /android/media/0/multirom ]] ||  mkdir -p $pref_path &&  mount /dev/mmcblk0p28 $pref_path
```

Comment:
Hi Marko,

all fixed I am sorry for the created mess... I am just waiting for sledges to come and see how can be fixed... since he does not want to close the request and create a clean new one. the new code will look something like this


        # If we are flashing boot partition and we are running multirom setup
        # which can be detected by having data_subdir= in th ecmdline skip boot
        # partition flashing as it is not needed.
        mrom_check=$(grep data_subdir= /proc/cmdline)
        if test $1 = boot &amp;&amp; [ ! -z "$mrom_check" ]; then
          echo "Skipping flashing of boot partition as multirom setup detected."
          echo "Update multirom boot image, ramdisk and kernel"
          pref_path="/android"
          [[ -d $pref_path/media/0/multirom ]] || mkdir -p $pref_path &amp;&amp; mount /dev/mmcblk0p28 $pref_path
          mrom_path=$(echo $mrom_check | sed 's/.*data_subdir=\(.*\)\/data/\1/')
          cp /boot/kernel-* $pref_path/$mrom_path/zImage
          cp /boot/boot-initramfs.gz $pref_path/$mrom_path/initrd.img
          cp /boot/hybris-boot.img $pref_path/$mrom_path/boot.img
          echo "Multirom update done"
          exit 0
        fi



there is still the issue of p28 which to be honest I do not know where I can get it for at the moment . 


Alin



---
Dr Alin Marin Elenahttp://alin.elena.space



---- On Sun, 23 Aug 2015 07:44:30 +0000 Marko Saukko &lt;notifications@github.com&gt; wrote ---- 

In sparse/usr/sbin/flash-partition:
 &gt; @@ -146,6 +146,14 @@ if on_device; then &gt; # partition flashing as it is not needed. &gt; if test $1 = boot &amp;&amp; grep data_subdir= /proc/cmdline &amp;&gt; /dev/null; then &gt; echo "Skipping flashing of boot partition as multirom setup detected." &gt; + echo "Update multirom boot image, ramdisk and kernel" &gt; + pref_path="/android" &gt; + [[ -d /android/media/0/multirom ]] || mkdir -p $pref_path &amp;&amp; mount /dev/mmcblk0p28 $pref_path  -d $pref_path/media/0/multirom or so right? Also extra spaces after || and &amp;&amp;
 —
Reply to this email directly or view it on GitHub.